app/views/home/support.html.erb

```diff
@@ -1,5 +1,47 @@
-<section class="support-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
+<section class="public-support-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
```

Comment:
- Remove the newsletter checkbox
- How can we help you? -> Increase the size of this box

![image](https://user-images.githubusercontent.com/8007722/72117440-b9a5b600-331b-11ea-957a-cf391a79093d.png)